lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -564,7 +564,7 @@ LogicalResult Deserializer::processExtInstImport(ArrayRef<uint32_t> words) {
   if (words.size() < 2) {
     return emitError(unknownLoc,
                      "OpExtInstImport must have a result <id> and a literal "
-                     "string for the extensed instruction set name");
+                     "string for the extented instruction set name");
```

Comment:
`extended` seems the right word here?

Reply:
Oh, you are right. I will address it in another PR.

Reply:
Ah, thank you for fixing it.